meta/Meta.cpp

```diff
@@ -6434,6 +6434,51 @@ void Meta::meta_sai_on_nat_event(
     }
 }
 
+void Meta::meta_sai_on_port_host_tx_ready_change(
+                    _In_ sai_object_id_t port_id,
+                    _In_ sai_object_id_t switch_id,
+                    _In_ sai_port_host_tx_ready_status_t host_tx_ready_status)
+{
+    SWSS_LOG_ENTER();
+
+    if (!sai_metadata_get_enum_value_name(
+            &sai_metadata_enum_sai_port_host_tx_ready_status_t,
+            host_tx_ready_status))
+    {
+        SWSS_LOG_WARN("port host_tx_ready value (%d) not found in sai_port_host_tx_ready_status_t. Dropping the notification",
+                host_tx_ready_status);
+
+        return;
+    }
+
+    auto ot = objectTypeQuery(port_id);
+    auto valid = false;
+    valid = (ot == SAI_OBJECT_TYPE_PORT);
+
+    if (!valid)
```

Comment:
used only once, you can put condition in te same line as if

Reply:
Done